src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterEnterOnTokenTests.cs

```diff
@@ -1347,6 +1347,112 @@ void Main(object o)
                 expectedIndentation: 12);
         }
 
+        [Trait(Traits.Feature, Traits.Features.SmartIndent)]
+        [WpfTheory]
+        [InlineData("x", "is < 7 and (>= 3 or > 50) or not <= 0;", 12)]
+        [InlineData("x is", "< 7 and (>= 3 or > 50) or not <= 0;", 12)]
+        [InlineData("x is <", "7 and (>= 3 or > 50) or not <= 0;", 12)]
+        [InlineData("x is < 7", "and (>= 3 or > 50) or not <= 0;", 12)]
+        [InlineData("x is < 7 and", "(>= 3 or > 50) or not <= 0;", 12)]
+        [InlineData("x is < 7 and (", ">= 3 or > 50) or not <= 0;", 12)]
+        [InlineData("x is < 7 and (>=", "3 or > 50) or not <= 0;", 12)]
+        [InlineData("x is < 7 and (>= 3", "or > 50) or not <= 0;", 12)]
+        [InlineData("x is < 7 and (>= 3 or", "> 50) or not <= 0;", 12)]
+        [InlineData("x is < 7 and (>= 3 or >", "50) or not <= 0;", 12)]
+        [InlineData("x is < 7 and (>= 3 or > 50", ") or not <= 0;", 12)]
+        [InlineData("x is < 7 and (>= 3 or > 50)", "or not <= 0;", 12)]
+        [InlineData("x is < 7 and (>= 3 or > 50) or", "not <= 0;", 12)]
+        [InlineData("x is < 7 and (>= 3 or > 50) or not", "<= 0;", 12)]
+        [InlineData("x is < 7 and (>= 3 or > 50) or not <=", "0;", 12)]
+        [InlineData("x is < 7 and (>= 3 or > 50) or not <= 0", ";", 12)]
+        public async Task IndentPatternsInLocalDeclarationCSharp9(string line1, string line2, int expectedIndentation)
```

Comment:
since the indentation is always the same, might want to take that out of hte inline data and make it more clearly part of the expected result.